iset.mm

```diff
@@ -104463,6 +104596,106 @@ which defines the set we are taking the supremum of must (a) be true at
+      XJUVEUXIUVESPZAUWRUVEUSVOZVNUXIUWRUVEVHOAUWRUVEVPVOVQVMVSUXNUVFSPZISPZUXB
+      UXRUYBVTUXJUYEUXKUXJUVEUYDUTZVMAUYFUXIUXKJWAAUXBUXIUXKUXFWAUVFIFWBWCWDZAU
+      XTUXIUXKAUXSHUXQMWEWAUXSUXPHUVFUXQUXOHBHUVFWFWIHUEUVFVABUXOBHUVFWGWHWJWKU
+      XNUVGUXOUVGUVFUXCPZUXNUXOGUXCUVFKUPUXNUXOUVFUWTPZUXORUYIUXNUYJUXOUXNUXRUY
+      JUYHUVFIFWLTWMBHUVFUWTHUWTWNWOWPWQWHWDUXJUXMRZUVGUVGUFZWRUXLUYKUYLUVGUYKU
+      XKUVGUXJUXMVRUXKUVFUXCGUYIUYJUXKBHUVFUWTURZIUVFVPZTKXAWSWTUVGXBXCUXJUXKVI
+      ZUXKUXMWRUXJUYFUYEUYOAUYFUXIJVMUYGIUVFXDXEUXKXLTXFAIUGZXGXHVLZUXHPZUYLNUY
+      QUOVEZUIZUYLNUBUEZUOVEZUIZUBUXHUJAUYPUXHPZUYRAVUDUWRUYPVHOZAIFVHOZVUEAUXA
+      VUFUXEIFVPTAIFAIJVNZAFUXFVNXIXJAUWRSPUYPSPZVUDVUEVTUXGAIJUTZUWRUYPYAXEWDU
+      WRUYPXKTAUYLNUYSAUVEUYSPZRZUVGUYPUYPUMOZVUKUVGRZVULUYQUYPVHOZVUMUYQUVEUYP
+      VUMUYPQPUYQQPVUMIAIQPVUJUVGVUGWAZXMZUYPXNTVUMUVEVUJUYCAUVGUYQUVEUSXOVNZVU
+      PVUJUYQUVEVHOAUVGUYQUVEVPXOVUMIUVEVUOVUQAUVGUXKVUJAUVGRUYJUXKUVGUYJAUYJUV
+      FUXCGUYMKXAVOUYNTXPXQXRVUMVUHVUHVULVUNVTAVUHVUJUVGVUIWAZVURUYPUYPXSXEWDVU
+      MUYPVUPXTYBWEVUCUYTUBUYQUXHVUAUYQVAUYLNVUBUYSVUAUYQUOYCYDYEXEYFAUWQUWJCSA
+      UWMUWEUWPUWIAUWDDUWLUVHAUCUWLUVHUCUEZUWLPZVUSUVHPZVTAVUSSPZVUSUGZGPZRZVUS
+      QPZVVDRZVUTVVAVVEVVGVVBVVFVVDVUSYGYHVVGVVBVVDVVGVVBVVCSPZVVDVVHVVFVVDVVCU
+      WTPZVVHVVIVVCUXCGBHVVCUWTURKXAIVVCUSTVOVVFVVBVVHVTZVVDVVFVUSYIPVVJVUSYJVU
+      SYKTVMWDVVFVVDVRVSYLUVGVVDNVUSSNUCYMUVFVVCGUVEVUSVBVCZYNUVGVVDNVUSQVVKYNY
+      OYPYQZYDAUWOUWHDQAUWNUWGUVAAUWFEUWLUVHVVLYRYSYTUUAUUBXJUWJCSQUUCUUDUVHQUN
+      AUVGNQUUEYPUUFUVRUWCCQUVKUVSUVQUWBUVBDUVJGUDUVJGUDUEZUVJPZVVMQPZVVMGPZUVI
+      UAVVMQURVVPVVMUWTPZVVOVVQVVMUXCGBHVVMUWTURKXAIVVMUUGTVVOVVNVVMUGZUVHPZVVR
+      UGZGPZVVPUVIVVSUAVVMQUAUDYMUVDVVRUVHUVCVVMVBVCUUHVVOVWAVVRQPZVWARVVSVVOVW
+      BVWAVVMUUIWMUVGVWANVVRQUVEVVRVAUVFVVTGUVEVVRVBVCYNWPVVOVVTVVMGVVOVVMVVMYJ
+      VDVCUUJUUMUUKZUULUVPUWADQUVOUVTUVLUVNEUVJGVWCUUNUUOUUPUUQUURUQ $.
+
+    $d A j m y $.  $d A m n $.  $d M j m $.  $d M m n $.  $d S a b m x y z $.
+    $d S j m x y z $.  $d S a m w x y z $.  $d a b m n ph $.
+    $d j m ph x y z $.
+    $( The infimum of a subset of an upper set of integers is less than or
+       equal to all members of the subset.  (Contributed by Jim Kingdon,
+       13-Jan-2022.) $)
+    infssuzledc $p |- ( ph -> inf ( S , RR , < ) <_ A ) $=
+      ( vx vy vz vb va cr clt wcel wbr wn cinf cv wa wb lttri3 adantl infssuzex
+      weq infclti cuz cfv crab elrabi eleq2s eluzelre 3syl inflbti mpd nltled )
+      ADPQUAZCAKLMNOPDQOUBZPRNUBZPRUCONUHVAVBQSTVBVAQSTUCUDAVAVBUEUFZABKLMCDEFG
+      HIJUGZUIACDRZCFUJUKZRZCPRIVGCBEVFULDBECVFUMHUNFCUOUPAVECUTQSTIAKLMNOPDCQV
+      CVDUQURUS $.
+
+    ${
+      $d S f g x y z $.  $d f g ph x y z $.
+      $( The infimum of a subset of an upper set of integers exists.  It should
+         be possible to show the infimum is an element of ` S ` , but this
+         theorem only shows it is an element of ` RR ` .  (Contributed by Jim
```

Comment:
Hopefully soon !

Reply:
In this case I think it probably will be a "soon" (depending a bit on how much I get distracted by going through the set.mm supremum/infimum sections more generally - which is all good stuff even aside from this particular topic).

Reply:
Making progress - #2440 . Although I did get a bit "distracted" by going through one supremum section, I have no regrets because a lot of this will be useful one way or another.

Reply:
I have a branch which addresses the "Hopefully soon" (in the sense of replacing `infuzre` with a similar theorem whose conclusion is `ph -> inf ( S , RR , < ) e. S`). But I'm waiting on submitting it until #2440 is reviewed because it depends on the contents of that pull request.

Reply:
Now submitted as #2443